private/rules/javadoc.bzl

```diff
@@ -107,7 +118,7 @@ javadoc = rule(
         "javadocopts": attr.string_list(
             doc = """javadoc options.
             Note sources and classpath are derived from the deps. Any additional
-            options can be passed here.
+            options can be passed here. If nothing is passed, a default list of options is used.
```

Comment:
Can you interpolate the list itself into this docstring?

Reply:
@jin I made your suggested change. I had to open a new PR due to constraints in my previous fork.
https://github.com/bazelbuild/rules_jvm_external/pull/1212